### ISSUE TITLE
Multiple warnings from make lesson-check

### ISSUE THREAD

**Issue by omsai** [swcarpentry/git-novice#721]
I'm creating a pull request for one of the open issues and was reading through how to create my local Jekyll rendering.  The [README.md](https://github.com/omsai/git-novice) file suggests reading [the lesson template documentation](https://carpentries.github.io/lesson-example/) which suggests running `make lesson-check` in step 4, but I it fails with:

```console
$ make lesson-check
./_episodes/01-basics.md: episode does not include "links.md"
./_episodes/02-setup.md: episode does not include "links.md"
./_episodes/02-setup.md:59: Unknown or missing blockquote type None
./_episodes/03-create.md: episode does not include "links.md"
./_episodes/04-changes.md:42: Unknown or missing code block type None
./_episodes/05-history.md: episode does not include "links.md"
./_episodes/06-ignore.md: episode does not include "links.md"
./_episodes/07-github.md: episode does not include "links.md"
./_episodes/08-collab.md: episode does not include "links.md"
./_episodes/09-conflict.md: episode does not include "links.md"
./_episodes/09-conflict.md:432: Unknown or missing code block type None
./_episodes/10-open.md: episode does not include "links.md"
./_episodes/11-licensing.md: episode does not include "links.md"
./_episodes/12-citation.md: episode does not include "links.md"
./_episodes/12-citation.md:51: Unknown or missing blockquote type None
./_episodes/13-hosting.md: episode does not include "links.md"
./_episodes/14-supplemental-rstudio.md: episode does not include "links.md"
./aio.md: Missing metadata entirely
./index.md: Root not set to "."
make: *** [Makefile:96: lesson-check] Error 1
```

# Steps to reproduce

```console
$ git clone git@github.com:omsai/git-novice.git
Cloning into 'git-novice'...
remote: Enumerating objects: 8322, done.
remote: Total 8322 (delta 0), reused 0 (delta 0), pack-reused 8322
Receiving objects: 100% (8322/8322), 9.81 MiB | 2.45 MiB/s, done.
Resolving deltas: 100% (5143/5143), done.
$ cd git-novice/
$ make lesson-check
Traceback (most recent call last):
  File "bin/lesson_check.py", line 565, in <module>
    main()
  File "bin/lesson_check.py", line 117, in main
    check_config(args.reporter, args.source_dir)
  File "bin/lesson_check.py", line 178, in check_config
    config = load_yaml(config_file)
  File "/home/omsai/src/git-novice/bin/util.py", line 164, in load_yaml
    return yaml.load(reader, Loader=yaml.FullLoader)
AttributeError: module 'yaml' has no attribute 'FullLoader'
make: *** [Makefile:96: lesson-check] Error 1
$ pipenv install -r requirements.txt 
Creating a virtualenv for this project…
Pipfile: /home/omsai/src/git-novice/Pipfile
Using /usr/bin/python3 (3.7.3) to create virtualenv…
⠦ Creating virtual environment...Already using interpreter /usr/bin/python3
Using base prefix '/usr'
New python executable in /home/omsai/.local/share/virtualenvs/git-novice-Y1Hom3dx/bin/python3
Also creating executable in /home/omsai/.local/share/virtualenvs/git-novice-Y1Hom3dx/bin/python
Installing setuptools, pip, wheel...
done.

✔ Successfully created virtual environment! 
Virtualenv location: /home/omsai/.local/share/virtualenvs/git-novice-Y1Hom3dx
Creating a Pipfile for this project…
Requirements file provided! Importing into Pipfile…
Pipfile.lock not found, creating…
Locking [dev-packages] dependencies…
Locking [packages] dependencies…
✔ Success! 
Updated Pipfile.lock (712390)!
Installing dependencies from Pipfile.lock (712390)…
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 2/2 — 00:00:01
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.
$ pipenv shell
Launching subshell in virtual environment…
. /home/omsai/.local/share/virtualenvs/git-novice-Y1Hom3dx/bin/activate
$ pip list
Package          Version
---------------- -------
pip              20.0.1 
PyYAML           5.3    
setuptools       45.1.0 
update-copyright 0.6.2  
wheel            0.33.6
$ make lesson-check
./_episodes/01-basics.md: episode does not include "links.md"
./_episodes/02-setup.md: episode does not include "links.md"
./_episodes/02-setup.md:59: Unknown or missing blockquote type None
./_episodes/03-create.md: episode does not include "links.md"
./_episodes/04-changes.md:42: Unknown or missing code block type None
./_episodes/05-history.md: episode does not include "links.md"
./_episodes/06-ignore.md: episode does not include "links.md"
./_episodes/07-github.md: episode does not include "links.md"
./_episodes/08-collab.md: episode does not include "links.md"
./_episodes/09-conflict.md: episode does not include "links.md"
./_episodes/09-conflict.md:432: Unknown or missing code block type None
./_episodes/10-open.md: episode does not include "links.md"
./_episodes/11-licensing.md: episode does not include "links.md"
./_episodes/12-citation.md: episode does not include "links.md"
./_episodes/12-citation.md:51: Unknown or missing blockquote type None
./_episodes/13-hosting.md: episode does not include "links.md"
./_episodes/14-supplemental-rstudio.md: episode does not include "links.md"
./aio.md: Missing metadata entirely
./index.md: Root not set to "."
make: *** [Makefile:96: lesson-check] Error 1
```

**Comment by kekoziar** [comment]
@omsai, did you create a PR for this issue?  I can't find it.

**Comment by omsai** [comment]
@kekoziar No, I never got past that error and don't see any other changes to my local repo.  Looking through my notes for around that time, it looks like I was teaching the git workshop with branches variant, so my best guess is it had something to do with that: https://carpentries-incubator.github.io/git-novice-branch-pr/

```console
$ git status
On branch gh-pages
Your branch is up to date with 'origin/gh-pages'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	Pipfile
	Pipfile.lock

nothing added to commit but untracked files present (use "git add" to track)
$ git log -2
commit ff8b73436131093244d0140bc1c87b4a8f1625da (HEAD -> gh-pages, origin/gh-pages, origin/HEAD)
Merge: 327f0b1 1b25bc4
Author: Madicken Munk <madicken.munk@gmail.com>
Date:   Wed Jan 15 16:53:13 2020 -0600

    Merge pull request #719 from jrkirk61/patch-1
    
    Make challenge solution Rstudio friendly

commit 1b25bc4d92b46405dfe22da7f03d287fcf75bc2b
Author: Jessica Holmes <kirk617@gmail.com>
Date:   Wed Jan 15 12:22:27 2020 -0600

    Make challenge solution Rstudio friendly
    
    Please see issue #718 for the reasoning of this request.
```

**Comment by kekoziar** [comment]
Thanks, @omsai.  I've investigated further, and recently the styles team updated the `lesson-check` to remove checking for `links.md` on lessons that use that style (https://github.com/carpentries/styles/commit/4df505249a27191c3475102cde4c150b0a131102).  So, there is no need to update and add `links.md` code.  Also, the code-block errors have since been fixed.

Closing.  `make lesson-check` runs with no output to fix lesson files.
